schema/types/cri_labels.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import "fmt"
+
+type CRILabels map[ACIdentifier]string
```

Comment:
@squeed Why labels keys are not arbitrary strings as well? Today, the [labels](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/labels.go#L34-L35) injected by kubelet is not ACIdentifier compliant. 
Although there is a TODO to change that...